test/ts/custom-elements.tsx

```diff
@@ -1,4 +1,4 @@
-import { createElement, Component, createContext } from '../../';
+import { createElement, Component, createContext, HTMLAttributes, MouseEventHandler } from '../../';
```

Comment:
`MouseEventHandler`, like `ContainerNode` above, seems to be a weird instance of global type access that shouldn't exist. I don't know why that worked, or what has since broke it, but I'd say this is correct?